src/core/resource.ts

```diff
@@ -48,29 +56,21 @@ export interface DataTemplate<S = {}> {
 type Status = 'LOADING' | 'FAILED';
 type InvalidatorMaps = { [key in SubscriptionType]: Map<string, Set<Invalidator>> };
 
-export function createTransformer<S, T>(template: DataTemplate<S>, transformer: TransformConfig<T, S>) {
-	return transformer;
-}
-
-function isAsyncResponse<S>(response: DataResponsePromise<S> | DataResponse<S>): response is DataResponsePromise<S> {
-	return (response as any).then !== undefined;
+function isAsyncResponse<S>(response: any): response is DataResponsePromise<S> {
+	return response.then !== undefined;
 }
 
-export function defaultFilter(query: ResourceQuery[], item: any) {
-	let filterValue = '';
-
-	query.forEach((q) => {
-		if (q.keys.indexOf('value') !== -1) {
-			filterValue = q.value || '';
+export function defaultFilter(queries: ResourceQuery[], item: any) {
```

Comment:
change the default filter to be generic and work on the query passed... the previous version was specific to the usage when it was created in widgets.